src/main/java/net/atos/zac/mail/MailService.java

```diff
@@ -200,13 +206,13 @@ private EnkelvoudigInformatieObjectData createDocumentInformatieObject(final Str
     private byte[] createPdfDocument(final String verzender, final String ontvanger, final String subject,
             final String body, final List<Attachment> attachments) {
         final ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
-        final Document document = new Document();
-        try {
-            PdfWriter.getInstance(document, byteArrayOutputStream);
-            document.open();
-            document.addTitle(subject);
-            final Font font = FontFactory.getFont(FontFactory.COURIER, 16, BaseColor.BLACK);
-            final Paragraph paragraph = new Paragraph(StringUtils.EMPTY, font);
+        try{
```

Comment:
Can we use the *try with resources* statement for the `Document`, `PdfWriter` and `PdfDocument` variables? That will auto-close these resources on exit of the try block (happy or exception flow).